### ISSUE TITLE
unable to update latest version 3.3.0 using installr

### ISSUE THREAD

**Issue by nandhinik2** [talgalili/installr#56]
![installr](https://cloud.githubusercontent.com/assets/18370741/15006342/7f995186-11ed-11e6-8b6a-fb255993c28a.PNG)
Kindly check the above issue and suggest

Thank you

**Comment by talgalili** [comment]
If you only see the option to upgrade to an older version of R, then change your mirror or try again in a few hours (it usually take around 24 hours for all CRAN mirrors to get the latest version of R).

**Comment by nandhinik2** [comment]
Thank you...

**Comment by talgalili** [comment]
R 3.3.0 is now available for Windows users as well. Simply run the following R code to upgrade:

install.packages("installr") # install 
setInternet2(TRUE)
installr::updateR() # updating R.

You can read more about this release here:
http://www.r-bloggers.com/r-3-3-0-is-released/

**Comment by ldecicco-USGS** [comment]
When I try this, I keep getting:

```
Error in setInternet2(TRUE) : use != NA is defunct
```
![image](https://cloud.githubusercontent.com/assets/1105215/15117739/8de726f2-15ce-11e6-86ee-f9313c62c344.png)

```
> devtools::session_info()
Session info ----------------------------------------------------------------------------
 setting  value                       
 version  R version 3.3.0 (2016-05-03)
 system   x86_64, mingw32             
 ui       RStudio (0.99.893)          
 language (EN)                        
 collate  English_United States.1252  
 tz       America/Chicago             
 date     2016-05-09                  

Packages --------------------------------------------------------------------------------
 package  * version date       source        
 devtools   1.11.1  2016-04-21 CRAN (R 3.3.0)
 digest     0.6.9   2016-01-08 CRAN (R 3.3.0)
 installr   0.17.5  2016-03-21 CRAN (R 3.3.0)
 memoise    1.0.0   2016-01-29 CRAN (R 3.3.0)
 withr      1.0.1   2016-02-04 CRAN (R 3.3.0)
```

any ideas?

**Comment by talgalili** [comment]
Thank you for reporting this @ldecicco-USGS 
It seems you already have R 3.3.0 installed (so no need to upgrade). But this bug will affect people in the future (once newer versions of R will come out). So I'll make sure to fix it before the next release.

With regards,
Tal

**Comment by talgalili** [comment]
The latest version of installr on github should now fix this issue for R 3.3.0 and forward.
